dcpy/library/templates/dob_jobapplications_parkingspaces.yml

```diff
@@ -0,0 +1,55 @@
+dataset:
+  name: dob_jobapplications_parkingspaces
+  acl: private
+  source:
+    url:
+      path: s3://edm-recipes/inbox/dob_bis_applications_parkingspots/dob_bis_applications_parkingspots.csv
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: null
+      type: NONE
+
+  destination:
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    sql: |
+      WITH numbered AS (
+        SELECT
+          ROW_NUMBER() OVER (PARTITION BY "job_#" ORDER BY DOBRunDate DESC) AS row_id,
+          "job_#" AS jobnumber,
+          DOBRunDate AS dobrundate,
+          J_ENCLOSED_NUM_SPACES AS enclosed_spaces,
+          cast(J_EX_PARKING_SPACES AS INTEGER) AS existing_parking_spaces,
+          cast(J_PR_PARKING_SPACES AS INTEGER) AS proposed_parking_spaces
+        FROM @filename
+        WHERE "doc_#" = '01'
+        AND ( 
+          "job_type" LIKE '%A1%' OR
+          "job_type" LIKE '%A2%' OR
+          "job_type" LIKE '%DM%' OR
+          "job_type" LIKE '%NB%'
+        )
+      )
+      SELECT 
+        jobnumber,
+        dobrundate,
+        enclosed_spaces,
+        existing_parking_spaces,
+        proposed_parking_spaces
+      FROM numbered
+      WHERE row_id = 1      
```

Comment:
This script is largely copied from the template `dob_jobapplications` - this is basically the same dataset, just with additional columns for parking. I figured it doesn't make sense to store it all, given that `jobapplications` is on opendata and theoretically could have updates (follow ups to old bis jobs), so will just keep this as a separate dataset with just keys (jobnum, rundate) and parking space info